README.md

```diff
@@ -32,3 +32,4 @@ Lines starting with `*` are automatically rendered as bullet points and phrases
 * Shaswat Datta [Shaswat27] (http://github.com/Shaswat27) 
 * Siddharth Kannan [icyflame](http://github.com/icyflame)
 * Srichandra [srichandra](https://github.com/srichandra)
+* Kenneth Simpson [knightshell] (https://github.com/knightshell301)
```

Comment:
You need to remove that ` ` between `[knightshell]` and `(https://github.com/knightshell301)` so that your github handle can be hyperlinked to the url.